src/Tables.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace Automattic\WP_CLI\SQLite;
+
+use WP_CLI;
+use PDO;
+use WP_SQLite_Translator;
+
+class Tables extends Base {
+
+	protected $translator;
+
+	public function __construct() {
+		$this->load_dependencies();
+		$this->translator = new WP_SQLite_Translator();
+	}
+
+	/**
+	 * Get the PDO instance.
+	 *
+	 * @return PDO
+	 */
+	protected function get_pdo() {
+		return $this->translator->get_pdo();
+	}
+
+	/**
+	 * Lists all tables in the SQLite database.
+	 *
+	 * @param array $assoc_args Associative array of options.
+	 * @return void
+	 */
+	public function run( $assoc_args = [] ) {
+		$pdo = $this->get_pdo();
+
+		// Get all tables
+		$stmt   = $pdo->query( "SELECT name FROM sqlite_master WHERE type='table' AND name NOT LIKE 'sqlite_%'" );
+		$tables = $stmt->fetchAll( PDO::FETCH_COLUMN );
+
+		// Remove system tables
+		$tables_to_exclude = array( '_mysql_data_types_cache' );
```

Comment:
As we are ignoring some tables as a part of the SQL query, would it make sense to do the same for this one?